cvat-ui/src/components/requests-page/styles.scss

```diff
@@ -0,0 +1,175 @@
+// Copyright (C) 2024 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+@import '../../base';
+
+.cvat-requests-scrollbar {
+    &::-webkit-scrollbar {
+        background-color: #fff;
+        width: 16px;
+    }
+
+    &::-webkit-scrollbar-track {
+        background-color: #fff;
+    }
+
+    &::-webkit-scrollbar-thumb {
+        background-color: #babac0;
+        border-radius: 16px;
+        border: 6px solid #fff;
+    }
+}
+
```

Comment:
Why did you decide to use vendor-specific css selectors?

Reply:
I believe we cant make pretty scrollbar with standard selectors. It works fine in chrome as its our main supported browser